### DISCUSSION TITLE
Windows support

### DISCUSSION THREAD

**Discussion by grigorii-horos** [lra/mackup#1970] [Ideas]
Hi. I use this backup application on Linux, mac and Windows. 
Yes, windows!

I use MSYS2 and it works really nice. I can backup files like `.zshrc`, and other `.*rc`. But when I create a new backup configuration for files from `AppData` these files appear on Linux in my home folder. But files from `~/Library` from my mac - no. I found that you use special condition for `~/Library` folder. Can you add the same condition for Windows for `AppData`?

https://github.com/lra/mackup/blob/master/mackup/utils.py#L371

**Comment by grigorii-horos** [comment]
Example of PR. Sorry, I'm not a Python developer, so please use my PR as template.

https://github.com/lra/mackup/pull/1954/files

**Comment by I-Want-ToBelieve** [comment]
Since the advent of Nix, the dotfile management of macOS and Linux has been put to an end by Home-Manager, but Nix does not support Windows and needs a tool like Mackup to fill the gap. 

Mackup is different from other dotfile management solutions in that it pre-defines the configuration file path of the software, which saves a lot of time, which is a killer feature that other dotfile management does not have, they are just fancy and deviate from the essence of dotfile management, saving time!
